include/deal.II/particles/particle_handler.h

```diff
@@ -696,6 +709,31 @@ namespace Particles
      */
     unsigned int handle;
 
+    /**
+     * A connection to the corresponding any_changes signal of the Triangulation
+     * which is attached to the particle_handler
+     */
+    boost::signals2::connection connection;
+
+    /** This variable stores a set from vertices to adjacent cells. It is used
```

Comment:
The test can go on the next line to be consistent, same with the comment for the next variable.